### PR TITLE
[Goonchem] Pipettes, Fixes, and Tweaks (again)

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Adds in Pipettes
pipettes
Huge thanks to Full of Skittles for the sprite!
- Pipettes are basically droppers that hold a single unit--they're largely for chem nerds or people who want to deal in very very precise amounts of reagents; can select 0.1 up to 1 units to transfer at a time, at 0.1 unit increments. 
- Chemistry has two, science testing lab has two, and the Virologist has one.

Reagent Changes
- Further updates more reagent's colors, descriptions, and reaction messages.
- Strange Reagent will now notify ghosts they're attempting to be revived in addition to giving a "failure--try again" message to the person doing the injection.
- Fixes the chemsprayer not spraying colored chemicals (the puffs would appear black, regardless of the chems it was spraying)
- Adds in Porktonium--doesn't do much unless OD'd on, then it can seriously kick your butt
 - Telebacon and the the chef's pig both have meat containing porktonium.
- Removes the Citalopram VIrus effect (this reagent doesn't exist anymore, and there's no equivalent)

Other
- Skeletons have NOCLONE now (can't revive them with SR and will require an upgraded cloner to clone someone who has been dead a long time).

Map
- Slightly alters science testing lab; they now have a box of syringes, 2 droppers, and 2 pipettes.

### CODE REVIEW

File: code/modules/reagents/newchem/food.dm [hunk 1]
```diff
@@ -142,4 +142,20 @@ datum/reagent/mugwort/on_mob_life(var/mob/living/M as mob)
...
+/datum/reagent/porktonium
+	name = "Porktonium"
+	id = "porktonium"
+	description = "A highly-radioactive pork byproduct first discovered in hotdogs."
+	reagent_state = LIQUID
+	color = "#AB5D5D"
+	metabolization_rate = 0.2
+
+datum/reagent/porktonium/overdose_process(var/mob/living/M as mob)
+	if(volume > 125)
```
Comment (Iamgoofball) [inline code/modules/reagents/newchem/food.dm]: You need to set overdose_threshold on the reagent define to trigger the overdose processing procs.

**Comment by Krausus** [comment]
Speaking of reagent fixes, could you fix [the sleeper's tier 3 chemicals](https://github.com/ParadiseSS13/Paradise/blob/d1932c0f64c512dbb9b4d470906475a55eba2f52/code/game/machinery/Sleeper.dm#L214)? Little typo:
```
list("epinephrine", "smorphine", "salbutamol", "styptic_powder", "oculine", "charcoal", "mutadone", "mannitol"))
```
`smorphine`

File: code/modules/reagents/newchem/food.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/modules/reagents/newchem/food.dm]: gotcha.

**Comment by Fox-McCloud** [comment]
@Krausus 

Will do

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 2]
```diff
@@ -692,7 +692,6 @@ datum
 			overdose_threshold = 200 // Hyperglycaemic shock
 
 			on_mob_life(var/mob/living/M as mob)
-				M.nutrition += 1*REM
```
Comment (Krausus) [inline code/modules/reagents/Chemistry-Reagents.dm]: Won't this make space twinkies and several candies non-nutritive? Not that it really makes sense for them to provide "nutrition", I suppose, but it does remove their actual value as food.

Comment (Fox-McCloud) [inline code/modules/reagents/Chemistry-Reagents.dm]: Yup; sugar's benefits are already pretty good, as is for a reagent that's so easy to acquire (and virtually unlimited)--not to mention that sugar, alone isn't very nutritious by itself; It's a good way to separate out "junk" food from nutritional food.

**Comment by Krausus** [comment]
* It's possible to get more regular droppers from nanomeds; should there be some way to get more pipettes?
* [Synthflesh](https://github.com/ParadiseSS13/Paradise/blob/93cf9de4bec8cb9cbe9a294bd8760a85a30a6179/code/modules/reagents/newchem/medicine.dm#L79-L95) only functions as an instant reaction, but leaves synthflesh in your system, which does nothing because it has no `on_mob_life`. Should probably just instantly remove itself.
* [Corgium](https://github.com/ParadiseSS13/Paradise/blob/93cf9de4bec8cb9cbe9a294bd8760a85a30a6179/code/modules/reagents/newchem/other.dm#L134-L153) and [Life](https://github.com/ParadiseSS13/Paradise/blob/93cf9de4bec8cb9cbe9a294bd8760a85a30a6179/code/modules/reagents/newchem/medicine.dm#L678-L687) also linger after reacting, despite having no function after their reaction occurs. Could their `result_amount` simply be set to 0 without breaking anything?
* Because patches are a `TOUCH` reaction, they have a chance to proc the 50% transfer of reagents [in the default `reaction_mob`](https://github.com/ParadiseSS13/Paradise/blob/93cf9de4bec8cb9cbe9a294bd8760a85a30a6179/code/modules/reagents/Chemistry-Reagents.dm#L29-L59), making patches sometimes transfer 150% of the reagents that they contain.

**Comment by Earthdivine** [comment]
....we still have Telebacon? I thought that was taken out ages ago.

**Comment by Fox-McCloud** [comment]
Having the reagents be in your system is consistent with other reagent's behavior, whether they do anything or not--there's literally goodness knows how many reagents that don't do anything in your system--editing them all so they deplete instantly isn't a good idea and hinder potential reactions...after all, what if someone want to make hairgrownium inside a group of people? With instantaneous depletion, that's not possible--it's not something that should be messed with either.

TOUCH will eventually be reworked which should correct the very issue you're mentioning here; currently all reagents have TOUCH = transfer to mob, which is ridiculously overpowered (with Baychem or Goonchem).

REM is meant to be a constant and to not vary---it's not a number that should ever really be tinkered with.

@Earthdivine 
It was re-added a little over a year ago; I have no idea why it was commented out to begin with; it just makes you a walking teleporter beacon for a short time-something easily accomplished by carrying around a beacon (and that's permanent).

**Comment by Krausus** [comment]
You wouldn't apply the same treatment to *all* reagents, because the rest of them don't *integrate directly with your flesh.* I hadn't considered the Hairgrownium reaction, though, so I suppose there's a use for it in your system after all. Corgium and Life still seem like they should be consumed in the reaction, seeing as there's other reactions (mostly pyrotechnic) that consume their reagent while producing an external effect.

As far as I can tell, Touch wasn't really a problem before we got patches, since they do their own transfer independent of the touch reaction's transfer. 150% patches should really be fixed before the eventual touch rework - it's a bug, and a nasty one if you're dealing with reagents with overdose thresholds. Getting screwed over with no logical explanation *sucks.*

**Comment by Fox-McCloud** [comment]
TOUCH has always been a problem as you've always been able to just walk up to a person and splash chloral or something-er-rather on the person. It's not realistic and it's not fun.

Patches functioning that way are not a bug; they're meant to apply the touch reaction immediately AND transfer the reagent to the person--this is a side effect of the fact that all reagents penetrate the skin---again, when that's reworked, it won't be a problem anymore.

**Comment by Fox-McCloud** [comment]
Corgi and Life reaction won't produce the reagent--that IS intended.

**Comment by Krausus** [comment]
> Patches functioning that way are not a bug

Bug, oversight, whatever. Semantics. The point was that magically duplicating reagents is clearly not working as intended. I guess it's fine to leave it unfixed for now, though.

**Comment by Fox-McCloud** [comment]
It's not a bug, and it's not an oversight; patches were always meant to proc TOUCH *and* transfer their full volume to the mob. 

What isn't intend is where *every* reagents getting multiplied (this is meant to be reserved for a few select reagents).

**Comment by Krausus** [comment]
> It's not a bug, and it's not an oversight; patches were always meant to proc TOUCH *and* transfer their full volume to the mob.

What. Of course that's not the bug. That's what patches do. The problem is--

> What isn't intend is where *every* reagent gets multiplied (this is meant to be reserved for a few select reagents).

-- that. Yes. The duplicating. Which was a problem with the default `on_mob_reaction` not having an exception for patches. The rework in #712 will fix that, though, so, problem solved.

A couple more things...
* Should Strange Reagent notify the ghosts of simple animals that get revived?
* It looks like Peridaxon (which is gone) is still required for [some surgery I've never heard of.](https://github.com/ParadiseSS13/Paradise/blob/26f11737659845e1d098af0c1b9c1057c257b3ba/code/modules/surgery/other.dm#L96-L168)

**Pull request merged**